docs/mapping/xml.md

```diff
@@ -6,7 +6,8 @@ format and comes with the following syntax to declare your uploadable fields:
 ```xml
 <!-- config/vich_uploader/Entity.Product.xml -->
 <!-- Attributes "mapping", "name" and "filename_property" are required -->
-<vich_uploader class="Acme\DemoBundle\Entity\Product">
+<vich_uploader xmlns="https://vich-uploader-bundle/schema/"
```

Comment:
So, I know that the "xmlns" attribute is not required, but I'm wondering how can the IDE understand where it's located (and so providing autocompletion). Could you clarify that?

Reply:
At least on phpstorm, if the file is within the project directories it will be picked up automatically.